src/localisation/package.xml

```diff
@@ -2,7 +2,7 @@
 <package>
   <name>localisation</name>
   <version>0.0.0</version>
-  <description>Snowbots Localisation Package</description>
+  <description>The ekf package</description>
```

Comment:
We should probably add a more detailed description here. Some points to note:
- This package contains both our in-house EKF and launch files for `robot_pose_ekf`
- What does EKF stand for? 
- Maybe a quick like to wikipedia too